apps/gov-website/pages/proposal/new.tsx

```diff
@@ -1,168 +1,86 @@
-						/>
-					</fieldset>
-
-					<fieldset className="mb-6">
-						<label
-							className="mb-1 block text-lg font-bold"
-							htmlFor="enacmentDelay"
-						>
-							Enactment Delay
-						</label>
-						<Select className="w-1/2" id="enacmentDelay">
-							<option value={24}>24 hours</option>
-							<option value={12}>12 hours</option>
-							<option value={6}>6 hours</option>
-							<option value={1}>1 hour</option>
-						</Select>
-					</fieldset>
-
-					<fieldset className="mb-6">
-						<label
-							htmlFor="functionCall"
-							className="flex cursor-pointer items-center"
-						>
-							<input
-								type="checkbox"
-								checked={functionCall}
-								onChange={onFunctionCallChange}
-								className="mr-1 inline-block"
-								id="functionCall"
-							/>
-							This proposal require a function call
-						</label>
-					</fieldset>
-
-					<If condition={functionCall}>
-						<h2 className="font-display border-hero mb-4 border-b-2 text-4xl uppercase">
-							Function Call
-						</h2>
-
-						<fieldset className="mb-6 grid grid-cols-2 gap-4">
-							<FunctionCallFieldSet.Provider extrinsics={extrinsics}>
-								<div className="cols-span-1">
-									<label className="mb-1 block text-lg font-bold">
-										Section
-									</label>
-									<FunctionCallFieldSet.Section />
-								</div>
-								<div className="cols-span-2">
-									<label className="mb-1 block text-lg font-bold">Method</label>
-									<FunctionCallFieldSet.Method />
-								</div>
-								<div className="col-span-full">
-									<label className="mb-1 block text-lg font-bold">
-										Arguments
-									</label>
-									<FunctionCallFieldSet.Args />
-								</div>
-							</FunctionCallFieldSet.Provider>
-						</fieldset>
-					</If>
-
-					<h2 className="font-display border-hero mb-4 border-b-2 text-4xl uppercase">
-						Sign and Submit
-					</h2>
-
-					<fieldset className="mb-6">
-						<label
-							className="mb-1 block text-lg font-bold"
-							htmlFor="enacmentDelay"
-						>
-							Signing Account
-						</label>
-						<AccountSelect required name="address" />
-					</fieldset>
-
-					<fieldset className="mt-16 text-center">
-						<Button type="submit" className="w-1/3 text-center">
-							<div className="flex items-center justify-center">
-								<span>Proceed</span>
-							</div>
-						</Button>
-						<p className="mt-2 text-sm">Estimated gas fee 2 CPAY</p>
-					</fieldset>
-				</form>
+				<h1 className="font-display text-hero mb-8 text-center text-7xl uppercase">
+					Submit a Proposal
+				</h1>
+
+				<p className="mb-8 text-lg">
+					To submit a proposal you must be a CENNZnet Councillor. Lorem laborum
+					dolor minim mollit eu reprehenderit culpa dolore labore dolor mollit
+					commodo do anim incididunt sunt id pariatur elit tempor nostrud nulla
+					eu proident ut id qui incididunt.
+				</p>
+				<FunctionCallFieldSet.Provider extrinsics={extrinsics}>
```

Comment:
nitpick, but by wrapping the entire `ProposalNewForm` in this provider, every change to the call fields will cause a rerender of the whole form

Reply:
yeah you would have thought so eh, I checked and it doesn't seem to affect it for some reason. Try put in `console.log("render")` in both form and fieldset provider, only the provider is rerendered when you switch to different section/method. Can you check on your side before I merge this!?

Reply:
oh sweet yeah that checks out